lib/src/json_schema/constants.dart

```diff
@@ -158,19 +158,28 @@ class SchemaVersion implements Comparable<SchemaVersion> {
   }
 }
 
-Map getJsonSchemaDefinitionByRef(String ref) {
+Map getStaticSchemaDefinitionById(String ref) {
```

Comment:
There's probably some cleanup that we could do here to support with and without the `#` fragment at the end, either path for vocabularies, in a more elegant way. For now this works :man_shrugging:

Reply:
~ `if (uri.host == refUri.host && uri.path == refUri.path)` and ignore fragment?

Reply:
Done in e1c0d18